app/controllers/proceeding_types/threshold_waivers_controller.rb

```diff
@@ -0,0 +1,57 @@
+module ProceedingTypes
+  class ThresholdWaiversController < ApplicationController
+    resource_description do
+      short 'Return threshold waivers'
+      formats(%w[json])
+      description <<-END_OF_TEXT
+      == Description
+        This endpoint accepts an array of proceeding types identified by#{' '}
+        and returns threshold waivers for the given proceeding types.
+
+          POST /proceeding_types/threshold_waivers
+
+      END_OF_TEXT
+    end
+
+    api :POST, 'proceeding_types/threshold_waivers', 'Create a request to retrieve a list of proceeding types with their threshold waivers'
+    param :request_id, :uuid, required: true, desc: 'A UUID generated by the client to uniquely identify this request.  It will be returned in the response.'
+    param :proceeding_types, Array, required: true, desc: 'CCMS codes of proceeding types in application'
+
+    returns code: :ok, desc: 'Successful response' do
+      property :request_id, :uuid, desc: 'The request_id specified in the request'
```

Comment:
As a result of changing to use the ccms_code, can we rename this parameter in the response to ccms_code for consistency, and to save confusion with the other field code, which doesn't get used anywhere.

Reply:
👍 Have made this change. Your comment also made me realise I hadn't updated some of the other parameter name changes in the APIPIE documentation so I have also fixed those